### PR TITLE
[10.x] Remove useless else statements

### PR DESCRIPTION
**Pull request by PerryvanderMeer**
Searched with regex, so this should be it for the whole repo.

### CODE REVIEW

**Review by driesvints — APPROVED** [review]

File: src/Illuminate/Database/Eloquent/Collection.php [hunk 1]
```diff
@@ -730,9 +730,9 @@ public function getQueueableRelations()
             return [];
         } elseif (count($relations) === 1) {
             return reset($relations);
-        } else {
-            return array_intersect(...array_values($relations));
         }
+
+        return array_intersect(...array_values($relations));
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Eloquent/Collection.php:735]: This change doesn't make sense without also changing the elseif.

File: src/Illuminate/Database/Console/ShowModelCommand.php [hunk 2]
```diff
@@ -165,9 +165,9 @@ protected function getVirtualAttributes($model, $columns)
                     return [Str::snake($matches[1]) => 'accessor'];
                 } elseif ($model->hasAttributeMutator($method->getName())) {
                     return [Str::snake($method->getName()) => 'attribute'];
-                } else {
-                    return [];
                 }
+
+                return [];
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Console/ShowModelCommand.php:170]: This change doesn't make sense without also changing the elseif.

File: src/Illuminate/Queue/QueueServiceProvider.php [hunk 3]
```diff
@@ -263,9 +263,9 @@ protected function registerFailedJobServices()
                 return $this->databaseUuidFailedJobProvider($config);
             } elseif (isset($config['table'])) {
                 return $this->databaseFailedJobProvider($config);
-            } else {
-                return new NullFailedJobProvider;
             }
+
+            return new NullFailedJobProvider;
```
Comment (GrahamCampbell) [inline src/Illuminate/Queue/QueueServiceProvider.php:268]: This change doesn't make sense without also changing the elseif.

File: src/Illuminate/View/Compilers/BladeCompiler.php [hunk 4]
```diff
@@ -340,12 +340,12 @@ public static function renderComponent(Component $component)
...
         } elseif ($view instanceof Htmlable) {
             return $view->toHtml();
-        } else {
-            return Container::getInstance()
-                ->make(ViewFactory::class)
-                ->make($view, $data)
-                ->render();
         }
+
+        return Container::getInstance()
```
Comment (GrahamCampbell) [inline src/Illuminate/View/Compilers/BladeCompiler.php:345]: This change doesn't make sense without also changing the elseif.

File: src/Illuminate/View/Concerns/ManagesComponents.php [hunk 5]
```diff
@@ -100,9 +100,9 @@ public function renderComponent()
                 return $view->with($data)->render();
             } elseif ($view instanceof Htmlable) {
                 return $view->toHtml();
-            } else {
```
Comment (GrahamCampbell) [inline src/Illuminate/View/Concerns/ManagesComponents.php:103]: This change doesn't make sense without also changing the elseif.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by driesvints** [comment]
Let's leave this be for now bc multiple prs have been closed in the past about this.

**Pull request closed without merging**